converter/internal/staticconvert/testdata/integrations.river

```diff
@@ -1,11 +1,9 @@
-prometheus.exporter.apache "integrations_apache_http" {
-	scrape_uri = "http://0.0.0.0/server-status?auto"
-}
+prometheus.exporter.agent "integrations_agent" { }
```

Comment:
git diff is weird here. These are new components and the apache components were simply moved down to line 25 shown below